### ISSUE TITLE
Limit for 10 packages

### ISSUE THREAD

**Issue by mcsakoff** [sdispater/mixology#9]
https://github.com/sdispater/mixology/blob/067c5aef14d7caeb733adff2b34e9489797f5113/mixology/version_solver.py#L68-L73
Is it intended limit or some testing code left over?

**Comment by pombredanne** [comment]
@mcsakoff good catch. I think this library is not updated at all and the canonical code lives instead in https://github.com/python-poetry/poetry/blob/master/poetry/mixology/version_solver.py#L67 instead.
@sdispater can you confirm this? Any intention to restore mixology as a self-standing library on your side?
(I need it for https://github.com/nexB/dependentcode/ ... so I would need to know what's your rough plan there, thanks!)

**Comment by pombredanne** [comment]
@mcsakoff note that was not a bug though, just a way to limit the number of resolution iterations to some number
